apps/web/pages/api/basenames/frame/frameResponses.ts

```diff
@@ -0,0 +1,129 @@
+import { getFrameMetadata, getFrameHtmlResponse } from '@coinbase/onchainkit/frame';
+import { FrameMetadataResponse } from '@coinbase/onchainkit/frame/types';
+import { isDevelopment } from 'apps/web/src/constants';
+
+export const DOMAIN = isDevelopment ? `http://localhost:3000` : 'https://www.base.org';
+
+export const initialFrame: FrameMetadataResponse = getFrameMetadata({
+  buttons: [
+    {
+      label: 'Claim',
+    },
+  ],
+  image: {
+    src: `${DOMAIN}/images/frames/basenames/initial-image.png`,
```

Comment:
can we import this instead of using public folder?

Reply:
tried this again and it's working locally, so let's run with this